src/process/process.rs

```diff
@@ -177,58 +176,47 @@ impl Process {
 		virtual_memory: &memory::VirtualMemory,
 	) -> ProcessResult<Percent> {
 		let memory_info = self.memory_info()?;
-		let percent = ((memory_info.rss() as f64 / virtual_memory.total() as f64) * 100.0) as f32;
-
-		Ok(percent)
+		let percent = (memory_info.rss() as f64 / virtual_memory.total() as f64) * 100.0;
+		Ok(percent as f32)
 	}
 
 	pub fn memory_percent_with_type(&self, r#type: MemType) -> ProcessResult<Percent> {
 		self.sys_memory_percent_with_type(r#type)
 	}
 
-	pub fn chidren(&self) {
-		self.sys_chidren()
+	pub fn children(&self) {
+		self.sys_children()
 	}
 
 	pub fn open_files(&self) -> ProcessResult<Vec<OpenFile>> {
 		self.sys_open_files()
 	}
 
 	pub fn connections(&self) {
-		self.sys_connections()
+		todo!()
 	}
 
-	pub fn connections_with_type(&self, r#type: NetConnectionType) {
-		self.sys_connections_with_type(r#type)
+	pub fn connections_with_type(&self, r#_type: NetConnectionType) {
+		todo!()
 	}
 
 	pub fn is_running(&self) -> bool {
-		match Process::new(self.pid) {
-			Ok(p) => p == *self,
-			Err(_) => false,
-		}
+		Process::new(self.pid).map(|p| p == *self).unwrap_or(false)
 	}
 
 	/// New method, not in Python psutil.
 	pub fn is_replaced(&self) -> bool {
-		match Process::new(self.pid) {
-			Ok(p) => p != *self,
-			Err(_) => false,
-		}
+		Process::new(self.pid).map(|p| p != *self).unwrap_or(false)
```

Comment:
I think this was actually clearer before, so maybe we could revert these ones.

Reply:
I found these to be clearer after the refactoring because they highlight the difference and similarities between `is_running` and `is_replaced` more, but if it hinders readability besides that, I can revert it, sure.

Reply:
I can see what you mean, but I think it was probably better before.